fenix/app/src/androidTest/java/org/mozilla/fenix/ui/ComposeTabbedBrowsingTest.kt

```diff
@@ -388,4 +392,46 @@ class ComposeTabbedBrowsingTest {
             verifyTurnOnSyncMenu()
         }
     }
+
+    @Test
+    fun privateModeStaysAsDefaultAfterRestartTest() {
+        val defaultWebPage = TestAssetHelper.getGenericAsset(mockWebServer, 1)
+
+        navigationToolbar {
+        }.enterURLAndEnterToBrowser(defaultWebPage.url) {
+        }.goToHomescreen {
+        }.togglePrivateBrowsingMode()
+        closeApp(composeTestRule.activityRule)
+        restartApp(composeTestRule.activityRule)
```

Comment:
nit: let's add a newline gap after `togglePrivateBrowsingMode()` and `homeScreen {`

Reply:
Done !